projects/canopy/src/lib/forms/validation/form.notes.ts

```diff
@@ -18,7 +15,7 @@ Import the module in your core component, e.g.:
 ~~~js
 @NgModule({
   ...
-  imports: [LgValidationModule]
+  imports: [..., LgValidationModule]
```

Comment:
Is this a mistake or some new voodoo I'm not aware of?

Reply:
It was done on purpose as most of the other stories have it.. it's just to say that a user might have more imports - it's just the notes file ;)